<ui version="4.0" >
 <class>HighscoreDialog</class>
 <widget class="QDialog" name="HighscoreDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>643</height>
   </rect>
  </property>
  <property name="palette" >
   <palette>
    <active>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>132</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>132</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>132</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>132</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>132</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>132</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle" >
   <string>Historie</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <widget class="QFrame" name="frame_2" >
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_2" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>9</y>
        <width>85</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>Rennen</string>
      </property>
      <property name="scaledContents" >
       <bool>false</bool>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QFrame" name="frame" >
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>9</y>
        <width>144</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>Hall of Fame</string>
      </property>
      <property name="scaledContents" >
       <bool>false</bool>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QTableWidget" name="tableWidgetLaps" >
     <property name="maximumSize" >
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QTableWidget" name="tableWidgetBestOf" />
   </item>
   <item row="1" column="0" >
    <widget class="QTableWidget" name="tableWidgetRaces" >
     <property name="minimumSize" >
      <size>
       <width>550</width>
       <height>250</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
